.travis.yml

```diff
@@ -29,7 +29,11 @@ before_install:
   - curl -L -O --insecure https://github.com/jedisct1/libsodium/releases/download/1.0.10/libsodium-1.0.10.tar.gz
   - tar -xf libsodium-1.0.10.tar.gz
   - (cd libsodium-1.0.10 && ./configure --prefix=$HOME/local && make && make install)
+  - curl -L -O --insecure https://github.com/dun/munge/releases/download/munge-0.5.13/munge-0.5.13.tar.xz
```

Comment:
Thanks for the review!

When testing a proposed PR, a failure is usually interpreted as a problem with the PR.  If (though unlikely) something changed in upstream munge that broke our master and hence derived PR's, it would be a confusing way to find out about it.

Munge has a pretty stable API, and I would be surprised if the author did not meticulously follow shared library versioning rules for any changes, so I'd say the more likely scenario for breakage is that munged options would change, and that would break tests, not the deployed product.

For those reasons, pinning it seems right to me.